kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -244,3 +263,106 @@ message EmptyDirMount {
     optional string medium = 3;
     optional string size_limit = 4;
 }
+
+// The proto messages below are copied from pipeline_spec.proto
+// and should match their structure.
+
+// Represents an input parameter. The value can be taken from an upstream
+// task's output parameter (if specifying `producer_task` and
+// `output_parameter_key`, or it can be a runtime value, which can either be
+// determined at compile-time, or from a pipeline parameter.
+message InputParameterSpec {

```

Comment:
Instead of duplicating this, let's try having it be in a separate file that both the main API and the Kubernetes platform API import.